packages/mongodb_atlas/data_stream/mongod_audit/fields/fields.yml

```
@@ -0,0 +1,63 @@
+- name: mongodb_atlas.mongod_audit
+  type: group
+  fields:
+    - name: hostname
+      type: keyword
+      description: Human-readable label that identifies the host that stores the log files that you want to download.
+    - name: local
+      type: group
+      fields:
+        - name: ip
+          type: ip
+          description: A document that contains the IP address of the running instance.
```

```suggestion
          description: IP address of the running instance.
```
Please replicate this change wherever we use document.